lib/modules/manager/asdf/extract.spec.ts

```diff
@@ -43,10 +43,12 @@ describe('modules/manager/asdf/extract', () => {
 
     it('can handle multiple tools in one file', () => {
       const res = extractPackageFile(
-        codeBlock`argocd 2.5.4
+        codeBlock`adr-tools 3.0.0
```

Comment:
```suggestion
        codeBlock`
adr-tools 3.0.0
```
`codeBlock` will trim the string